src/components/network-chart/index.js

```diff
@@ -295,6 +301,13 @@ class NetworkChartImpl extends React.PureComponent<Props> {
           >
             <VirtualList
               className="treeViewBody"
+              ariaRole="listbox"
+              ariaLabel="list"
```

Comment:
This label will be exposed as a string to the screen readers. I think we need something more descriptive. This could be "List of all network markers" for example.